src/Analyzers/CSharp/CodeFixes/UseUtf8StringLiteral/UseUtf8StringLiteralCodeFixProvider.cs

```diff
@@ -48,12 +48,26 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
+            var readOnlySpanType = semanticModel.Compilation.GetBestTypeByMetadataName(typeof(ReadOnlySpan<>).FullName!);
+            // The analyzer wouldn't raise a diagnostic if this were null
+            Contract.ThrowIfNull(readOnlySpanType);
+
             foreach (var diagnostic in diagnostics)
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
                 var node = diagnostic.Location.FindNode(getInnermostNodeForTie: true, cancellationToken);
-                var stringValue = GetUtf8StringValueForDiagnostic(semanticModel, diagnostic, cancellationToken);
+                var arrayOp = GetArrayCreationOperation(semanticModel, diagnostic, cancellationToken);
+                Contract.ThrowIfNull(arrayOp.Initializer);
```

Comment:
is this also checked in the analyzer?

Reply:
Yes. Also, the contract check just moved, it was here previously too